config/aws_config.py

```
@@ -0,0 +1,41 @@
+MAX_RETRIES = 3
+TIMEOUT_SECONDS = 30
+
+# Email Configuration (if using SES like bk-feeder)
+SES_FROM_EMAIL = os.getenv('SES_FROM_EMAIL', '')
+SES_TO_EMAILS = os.getenv('SES_TO_EMAILS', '').split(',')
+
+# PACER Configuration (for future integration)
+PACER_USERNAME = os.getenv('PACER_USERNAME', '')
+PACER_PASSWORD = os.getenv('PACER_PASSWORD', '')
+
+def get_aws_session():
```

Ruff (0.11.9)

Missing return type annotation for public function `get_aws_session`

(ANN201)